pkg/operation/botanist/component/istio/istiod.go

```diff
@@ -99,7 +99,7 @@ func (i *istiod) Deploy(ctx context.Context) error {
 	}
 
 	// TODO(mvladev): Rotate this on on every istio version upgrade.
```

Comment:
nit
```suggestion
	// TODO(mvladev): Rotate this on every istio version upgrade.
```

Reply:
Removed "on" with the latest (amended) commit.